tests/microos/rollback.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test downgrading to previous system version after
```

Comment:
I think this is not downgrade, it is better to change it to 'rollback'. How about 'Test rollback to original system ...'

Reply:
yes, rollback is to return to previous state, it is more clear, upgrade and downgrade are ambiguous terms.

Reply:
Done